libImaging/Resample.c

```diff
@@ -107,7 +107,7 @@ ImagingPrecompute(int inSize, int outSize, struct filter *filterp,
     if ( ! kk)
         return 0;
 
-    xbounds = calloc(outSize * 2, sizeof(int));
+    xbounds = malloc(outSize * 2 * sizeof(int));
```

Comment:
I've returned malloc anywhere and clearing values [with loop](https://github.com/uploadcare/pillow-simd/blob/12c8cf94f244b3fc1039e71c7a664b97a2045554/libImaging/Resample.c#L138).